### ISSUE TITLE
When i run this code ,get loss is negative??

### ISSUE THREAD

**Issue by YoudianguaiYi** [milesial/Pytorch-UNet#78]
Starting epoch 10/10.
0.0000 --- loss: -1086.891846
0.0526 --- loss: -2474.710449
0.1053 --- loss: -2207.034180
0.1579 --- loss: -1271.199463
0.2105 --- loss: -2474.710449
0.2632 --- loss: -1237.979980
0.3158 --- loss: -1086.891846
0.3684 --- loss: -1271.199463
0.4211 --- loss: -2196.364502
0.4737 --- loss: -2196.364502
0.5263 --- loss: -2207.034180
0.5789 --- loss: -1695.583130
0.6316 --- loss: -3084.041992
0.6842 --- loss: -3084.041992
0.7368 --- loss: -1925.202637
0.7895 --- loss: -1605.029907
0.8421 --- loss: -1695.583130
0.8947 --- loss: -1237.979980
0.9474 --- loss: -1605.029907
Epoch finished ! Loss: -1980.159640842014
Validation Dice Coeff: 1.9720964431762695
Checkpoint 10 saved !

Is anyone have the same problem???

**Comment by milesial** [comment]
`nn.BCELoss()` shouldn't be negative. How are you loading your masks? Have you pulled the latest commit?

**Comment by kidliet** [comment]
please ensure mask is 0-1，not 0-255. because the loss function is BCELoss.

**Comment by YoudianguaiYi** [comment]
> please ensure mask is 0-1，not 0-255. because the loss function is BCELoss.

OK, thanks, i have changed the code and get right result!

**Comment by YoudianguaiYi** [comment]
> nn.BCELoss() shouldn't be negative. How are you loading your masks? Have you pulled the latest commit?

OK, thanks, i have changed the code and get right result!
